src/vaadin-contextmenu-items-mixin.html

```diff
@@ -9,7 +10,7 @@
      * @memberof Vaadin
      * @extends Vaadin.ItemElement
      */
-    class ContextMenuItemElement extends Vaadin.ItemElement {
+    class ContextMenuItemElement extends Vaadin.DirMixin(Vaadin.ItemElement) {
```

Comment:
`vaadin-item` already extends `DirMixin`: https://github.com/vaadin/vaadin-item/blob/4d361148606871f43a42c46343ed5529cb78d78e/src/vaadin-item.html#L68.
No need to wrap it over here.

Reply:
Done. I added it first because I hadn't update the `vaadin-item` dependency and it didn't have the `DirMixin` yet. Thanks!